docs/apm/service-overview.asciidoc

```diff
@@ -2,4 +2,106 @@
+potential issues, select a <<services,*service*>> to bring you to the *Service overview* page.
+
+[discrete]
+[[service-latency]]
+=== Latency
+
+Response times for the service. You can filter the *Latency* chart to display the average,
+95th, or 99th percentile latency times for the service.
+
+[role="screenshot"]
+image::apm/images/latency.png[Service latency]
+
+[discrete]
+[[service-traffic-transactions]]
+=== Traffic and transactions
+
+The *Traffic* chart visualizes the average number of transactions per minute for the selected service.
+
+The *Transactions* table displays a list of _transaction groups_ for the
+selected service and includes the latency, traffic, error rate, and the impact for each transaction.
+Transactions that share the same name are grouped, and only one entry is displayed for each group.
+
+By default, transaction groups are sorted by _Impact_ to show the most used and slowest endpoints in your
+service. If there is a particular endpoint you are worried about, click *View transactions* to view the
+<<transactions, transactions overview>>.
+
+[role="screenshot"]
+image::apm/images/traffic-transactions.png[Traffic and transactions]
+
+[discrete]
+[[service-error-rates]]
+=== Error rate and errors
+
+The *Error rate* chart displays the average error rates relating to the service, within a specific time range.
+
+The *Errors* table provides a high-level view of each error message when it first and last occurred,
+along with the total number of occurrences. This makes it very easy to quickly see which errors affect
+your services and take actions to rectify them. To do so, click *View errors*.
+
+[role="screenshot"]
+image::apm/images/error-rate.png[Error rate and errors]
+
+[discrete]
+[[service-span-duration]]
+=== Span types average duration and dependencies
+
+The *Average duration by span type* chart visualizes each span type's average duration and helps you determine
+which spans could be slowing down transactions. The "app" label displayed under the
+chart indicates that something was happening within the application, which could signal that the
+agent does not have auto-instrumentation for whatever was happening during that time.
+
+The *Dependencies* table displays a list of downstream services or external connections relevant
+to the service at the selected time range. The table displays latency, traffic, error rate, and the impact of
+each dependency. By default, dependencies are sorted by _Impact_ to show the most used and the slowest dependency.
+If there is a particular dependency you are worried about, click *View service map* to view the related
+<<service-maps, service map>>.
+
+[role="screenshot"]
+image::apm/images/spans-dependencies.png[Span type duration and dependencies]
+
+[discrete]
+[[service-instances]]
+=== All instances
+
+The *All instances* table displays a list of all the available service instances within the selected time range.
```

Comment:
Might be worth noting that the instance could be a host or a container depending on how the service runs, and that clicking the node name will bring you to the metrics tab for the service that shows more detailed charts.

Reply:
Thanks for the additional wording. I'll add it.